frontend/Dockerfile.frontend

```diff
@@ -0,0 +1,15 @@
+FROM node:18
+
+WORKDIR /app
+
+COPY . .
+
+RUN corepack enable
+
+RUN corepack prepare yarn@stable --activate
+
+RUN yarn install
+
+CMD ["yarn", "run", "dev"]
```

Comment:
We should run `yarn build` and have the startup command be `yarn preview --host --port 4173` instead (exposing `4173` in Docker Compose instead)

Ideally, we should use the Node.js adapter or SSG + nginx in SvelteKit, but that requires a bit more work and `preview` seems fine for the time being for small-scale production use